components/Workshop/WorkshopListingPastSlotList.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import { Box, Heading, Stack } from "@chakra-ui/react";
+import { Booking, Slot } from "@schemas";
+import { WorkshopListingPastSlot } from "@components/Workshop/WorkshopListingPastSlot";
+import Show from "@components/Helpers/Show";
+import NoResults from "@components/NoResults";
+
+interface IProps {
+  slots: Slot[];
+  bookings: Booking[];
+}
+
+export default function WorkshopListingPastSlotList({
+  slots,
+  bookings,
+}: IProps) {
+
+
+  const getActiveBookingsForSlot = (slot: Slot): Booking[] => {
+    return bookings.filter((b) => b.slot_id === slot.id);
+  };
+
+
+  return (
+    <Box
+      borderBottomWidth={"1px"}
+      borderBottomColor="gray.200"
+      p="6"
+      rounded="md"
+      px={{ base: "6", md: "16" }}
+    >
+      <Stack>
+        <Heading as="h2" size="md" pb="0.5em">
+          Past slots
```

Comment:
Maybe we should title this "Past Sessions". I know that we use slots in the database schema, but maybe a session makes more sense from the UX perspective. What do you think? If we go with this, then we can rename "Availability" to "Upcoming Sessions" on the `WorkshopListingUpcomingSlotList`.

See also a separate, top level comment about refactoring the two session types into a tab based layout.

Reply:
I agree, "sessions" instead of "slots" seems generally clearer. I will modify that.